source/includes/fact-atlas-roles.rst

```diff
@@ -0,0 +1,14 @@
+The user specified in the connection string must have the
+:atlasrole:`atlasAdmin` role.
+
+To use ``mongosync`` in the reverse direction, you must `create a
```

Comment:
We should phrase this more as "IF" they want to run in reverse, then you must create the Atlas custom role.

Reply:
link to the new /reverse page

Reply:
The docs normally follow the pattern: 

"To do X, take this action. "

It's more direct than saying, "If you want to do X, take this action" which helps the reader.

I've created a [ticket](https://jira.mongodb.org/browse/DOCSP-23084) to link to the /reverse page when it's available.